### PR TITLE
v01

### PR DESCRIPTION
**Pull request by katerynashylina**
- [DEMO LINK](https://katerynashylina.github.io/layout_catalog/)
- [TEST REPORT LINK](https://katerynashylina.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by loralevitska — CHANGES_REQUESTED** [review]
HI) Where are your style files. Please push all solution to github.
image
Also pay attention to your failed tests. Check all data attributes
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

**Review by agniya-i — CHANGES_REQUESTED** [review]
Try to fix failed tests, if you have any questions, please ask them in fe_chat

File: src/styles/card.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
+.cards {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: (4 * $card-width) + (3 * $card-margin-right) + (2 * $padding-horizont);
```
Comment (SanyaBratashchuk) [inline src/styles/card.scss:7]: Do not set external geometry directrly to the BEM block

File: src/styles/card.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  background-color: #{$white-color};
+  border: 1px solid #{$like-white-color};
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-bottom: $card-margin-top;
```
Comment (SanyaBratashchuk) [inline src/styles/card.scss:22]: same external geometry. Use BEM mixes

File: src/styles/normalize.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+$white-color: #fff;
+$like-white-color: #f3f3f3;
+$dark-color: #060b35;
+$grey-color: #616070;
+
+$card-width: 200px;
+$card-margin-top: 46px;
+$card-margin-right: 48px;
+$padding-horizont : 40px;
+$padding-vertical : 50px;
```
Comment (SanyaBratashchuk) [inline src/styles/normalize.scss:11]: Move variables to the separate file

File: src/styles/card.scss (hunk 2 above)
Comment (DarkMistyRoom) [reply src/styles/card.scss:11]: margin is still here
image

File: src/styles/container.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: (4 * $card-width) + (3 * $card-margin-right) + (2 * $padding-horizont);
+  padding: $padding-vertical $padding-horizont;
+  margin: 0 auto;
```
Comment (DarkMistyRoom) [inline src/styles/container.scss:9]: Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

File: src/styles/header.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-weight: 500;
+}
+
+.header__logo {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px 0;
+}
```
Comment (DarkMistyRoom) [inline src/styles/header.scss:15]: use nesting

File: src/styles/stars.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+}
+  
+.stars--0 .stars__star:nth-child(n + 1),
+.stars--1 .stars__star:nth-child(n + 2),
+.stars--2 .stars__star:nth-child(n + 3),
+.stars--3 .stars__star:nth-child(n + 4),
+.stars--4 .stars__star:nth-child(n + 5),
+.stars--5 .stars__star:nth-child(n + 6) {
+  background-image: url("../images/star.svg");
+}
```
Comment (DarkMistyRoom) [inline src/styles/stars.scss:21]: image

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Pull request closed without merging**